DOCUMENTRETRIEVAL/documentRetrieval/library/documentRetrieval.py

```diff
@@ -276,14 +276,14 @@ def tfidf_sum_weight(probability,  token_frequency, n, idf, word, alpha, origina
     return tfidf_value
 
 
-def tfidf_score_str(tokens,texts,tfidf_function_name,m = 10,*args):
+def tfidf_score_str(tokens,texts,tfidf_function_name,nb_all_texts_in_db, m = 10,*args):
```

Comment:
Function parameters should be separated by a space.

`def tfidf_score_str(tokens, texts, tfidf_function_name, nb_all_texts_in_db, m=10, *args)`

No spaces on either side of the equal sign: `m = 10` -> `m=10`

What does `nb` mean in the `nb_all_texts_in_db`?